migrations/mysql/2024-09-04-091351_use_device_types/down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `twofactor_incomplete` DROP COLUMN `device_atype`;
```

Comment:
```suggestion
ALTER TABLE `twofactor_incomplete` DROP COLUMN `device_type`;
```
Same name change for all the other migration files